sdk/samples/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -68,6 +68,8 @@ AZ_NODISCARD az_keyvault_keys_client_options az_keyvault_keys_client_options_def
   options.retry.retry_delay_msec = 1 * _az_TIME_MILLISECONDS_PER_SECOND;
   options.retry.max_retry_delay_msec = 30 * _az_TIME_MILLISECONDS_PER_SECOND;
 
+  options._internal._telemetry_options = _az_http_policy_telemetry_options_default();
```

Comment:
Can we move the code around to this:
az_keyvault_keys_client_options options = (az_keyvault_keys_client_options){
    ._internal = { .api_version = _az_http_policy_apiversion_options_default(), },
    .retry = _az_http_policy_retry_options_default(),
    ._telemetry_options = _az_http_policy_telemetry_options_default(),  <-- move this here
  };

Reply:
looks like it was already there for storage.
So, only updating keyvault with this